testing/trino-testing/src/main/java/io/trino/testing/AbstractTestDistributedQueries.java

```diff
@@ -71,6 +73,28 @@
 public abstract class AbstractTestDistributedQueries
         extends AbstractTestQueries
 {
+    @DataProvider
+    public static Object[][] viewKinds()
+    {
+        return Arrays.stream(ViewKind.values()).map(kind -> new Object[] {kind}).toArray(Object[][]::new);
+    }
+
+    private enum ViewKind
```

Comment:
I would simply use String. This structure seems to be a bit too elaborated, or you can pass two parameters with `@DataProvider`.

Reply:
You're right, I converted it to two parameters, and the data providers is just creating an array directly.